core/settings.go

```diff
@@ -260,6 +261,7 @@ func (c S3Config) Validate() error {
 		validation.Field(&c.Region, validation.When(c.Enabled, validation.Required)),
 		validation.Field(&c.AccessKey, validation.When(c.Enabled, validation.Required)),
 		validation.Field(&c.Secret, validation.When(c.Enabled, validation.Required)),
+		validation.Field(&c.ForcePathStyle, validation.When(c.Enabled, validation.Required)),
```

Comment:
This line could be removed. The `ForcePathStyle` should be optional, not required.